File: JavaScript/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

linha1 = lines.shift().split(" ")
linha2 = lines.shift().split(" ")

cod_p1 = linha1[0]
n_p1 = linha1[1]
valor_p1 = linha1[2]

cod_p2 = linha2[0]
n_p2 = linha2[1]
valor_p2 = linha2[2]

res_p1 = n_p1 * valor_p1
res_p2 = n_p2 * valor_p2
resultado = res_p1 + res_p2

console.log(`VALOR A PAGAR: R$ ${resultado.toFixed(2)}`)

File: C/1010.c
#include <stdio.h>

int main() {

  int cp1, np1, cp2, np2;
  double vp1, vp2, total;

  scanf("%d", &cp1);
  scanf("%d", &np1);
  scanf("%lf", &vp1);

  scanf("%d", &cp2);
  scanf("%d", &np2);
  scanf("%lf", &vp2);

  total = (np1 * vp1) + (np2 * vp2);

  printf("VALOR A PAGAR: R$ %.2lf\n", total);
  
  return 0;
  
}
